Sources/MapboxNavigation/VisualInstruction.swift

```diff
@@ -85,10 +85,11 @@ extension VisualInstruction {
     /// Returns whether the `VisualInstruction`’s maneuver image should be flipped according to the driving side.
     public func shouldFlipImage(side: DrivingSide) -> Bool {
```

Comment:
Incidentally, this method is contained in an `#if canImport(CarPlay)` but its only call site is not. 😳 As tail work, I guess we should remove all the `#if canImport(CarPlay)`s now that the SDK requires a new enough Xcode to build. That will also eliminate a jazzy USR warning.